e2e/app/component/navigation.ts

```diff
@@ -1,6 +1,6 @@
 import {ElementHandle, Page} from 'puppeteer';
 import {clrIconXpath} from 'app/element/xpath-defaults';
-import {findIcon} from 'app/element/xpath-finder';
+import ClrIconLink from 'app/element/clr-icon-link';
```

Comment:
What does the CLR stand for? I don't think we necessarily have to reuse names that are confusing in the main app.